components/cache.rst

```diff
@@ -21,11 +25,27 @@ You can install the component in 2 different ways:
 * :doc:`Install it via Composer </components/using_components>` (``symfony/cache`` on `Packagist`_);
 * Use the official Git repository (https://github.com/symfony/cache).
 
-Key Concepts
-------------
+Cache (PSR-6) Versus Simple Cache (PSR-16)
+------------------------------------------
```

Comment:
@nicolas-grekas Is this small section accurate? Everything else is straightforward - I just wanted you to check this one spot! Thanks!